handler/oauth2/flow_refresh.go

```diff
@@ -30,6 +29,11 @@ type RefreshTokenGrantHandler struct {
 		fosite.AudienceStrategyProvider
 		fosite.RefreshTokenScopesProvider
 	}
+
+	// IgnoreRequestedScopeNotInOriginalGrant determines the action to take when the requested scopes in the refresh
+	// flow were not originally granted. If false which is the default the handler will automatically return an error.
+	// If true the handler will filter out / ignore the scopes which were not originally granted.
+	IgnoreRequestedScopeNotInOriginalGrant bool
```

Comment:
Side note when looking at this. I have a per-client implementation of this utilizing an interface. Maybe this is more desirable?

Reply:
Why is this not made through a new config interface?

Reply:
To me it looks like currently all config is done through config interface, even if it is not meant for hot reloading?